mm2src/coins/eth.rs

```diff
@@ -1481,7 +1482,7 @@ impl WatcherOps for EthCoin {
             .watcher_reward
             .clone()
             .ok_or_else(|| ValidatePaymentError::WatcherRewardError("Watcher reward not found".to_string())));
-        let expected_reward_amount = try_f!(wei_from_big_decimal(&watcher_reward.amount, self.decimals));
+        let expected_reward_amount = try_f!(wei_from_big_decimal(&watcher_reward.amount, 18));
```

Comment:
Could we use a constant name instead of 18? I guess it is ETH_DECIMALS

Reply:
Good note, will do shortly.

Reply:
@dimxy Done.